src/main/java/au/org/emii/ncdfgenerator/NcdfDefinitionXMLParser.java

```diff
@@ -1,276 +1,374 @@
 
 package au.org.emii.ncdfgenerator;
 
-import au.org.emii.ncdfgenerator.IVariableEncoder;
-import au.org.emii.ncdfgenerator.VariableEncoder;
 
+import java.util.List;
 import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.AbstractMap.SimpleImmutableEntry;
 
+import java.sql.Connection;
+import java.io.InputStream;
+
+import org.w3c.dom.Document;
 import org.w3c.dom.Node;
 import org.w3c.dom.NamedNodeMap;
 
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.DocumentBuilder;
+
+import java.lang.Exception ;
+
+import au.org.emii.ncdfgenerator.IDimension;
+import au.org.emii.ncdfgenerator.NcdfGeneratorException;
 
-class NcdfDefinitionXMLParser
-{
-	private boolean isNodeName( Node node, String name )
-	{
-		return node.getNodeType() == Node.ELEMENT_NODE
-			&& node.getNodeName().equals( name );
-	}
 
-	private String nodeVal( Node node  )
+class Helper
```

Comment:
I wanted to nest it within the NcdfDefinitionXMLParser to indicate the scope of the helper a bit better. However javac complains  that nested classes can't have static methods. As an alternative, I thought about giving it a longer name, but then decided it made use at the call site overly verbose.